package.json

```diff
@@ -61,7 +61,8 @@
     "tslint": "5.4.3",
     "tslint-microsoft-contrib": "5.0.0",
     "typescript": "2.9.2",
-    "webpack": "^3.8.1"
+    "webpack": "^3.8.1",
```

Comment:
Please note I've also update versions of other package. Please update the same packages as well